common-lib/src/main/scala/com/gu/workflow/util/AWS.scala

```diff
@@ -1,31 +1,39 @@
 package com.gu.workflow.util
 
 import com.amazonaws.auth.profile.ProfileCredentialsProvider
-import com.amazonaws.auth.{AWSCredentialsProviderChain, DefaultAWSCredentialsProviderChain, EnvironmentVariableCredentialsProvider, InstanceProfileCredentialsProvider}
+import com.amazonaws.auth.{AWSCredentialsProviderChain, InstanceProfileCredentialsProvider}
 import com.amazonaws.regions.{Region, Regions}
-import com.amazonaws.services.cloudwatch.{AmazonCloudWatchAsyncClient, AmazonCloudWatchAsyncClientBuilder}
-import com.amazonaws.services.dynamodbv2.{AmazonDynamoDBClient, AmazonDynamoDBClientBuilder}
 import com.amazonaws.services.dynamodbv2.document.DynamoDB
-import com.amazonaws.services.ec2.{AmazonEC2Client, AmazonEC2ClientBuilder}
+import com.amazonaws.services.dynamodbv2.{AmazonDynamoDB, AmazonDynamoDBClientBuilder}
 import com.amazonaws.services.ec2.model.{DescribeTagsRequest, Filter}
+import com.amazonaws.services.ec2.{AmazonEC2, AmazonEC2ClientBuilder}
 import com.amazonaws.util.EC2MetadataUtils
 
 import scala.collection.JavaConverters._
 
 object AWS {
+  lazy val credentialsProvider = new AWSCredentialsProviderChain(
+    new ProfileCredentialsProvider("workflow"),
+    InstanceProfileCredentialsProvider.getInstance()
+  )
 
   lazy val region: Region = Region getRegion Regions.EU_WEST_1
 
-  lazy val EC2Client = AmazonEC2ClientBuilder.standard.withRegion(region.getName).build
-  lazy val CloudWatch = AmazonCloudWatchAsyncClientBuilder.standard.withRegion(region.getName).build
-  lazy val DynamoDb = AmazonDynamoDBClientBuilder.standard.withRegion(region.getName).withCredentials(
-    new AWSCredentialsProviderChain(
-      new EnvironmentVariableCredentialsProvider(),
-      InstanceProfileCredentialsProvider.getInstance(),
-      new ProfileCredentialsProvider("workflow"),
-      new DefaultAWSCredentialsProviderChain
```

Comment:
We're [using an InstanceProfile](https://github.com/guardian/editorial-tools-platform/blob/master/cloudformation/workflow-account/workflow/workflow-frontend.yaml#L326), so provider chain [can become a lot simpler](https://github.com/guardian/workflow-frontend/pull/216/files#diff-2b19a54750b96913359e3934504c2b0dR15).

Reply:
What about running locally? Shouldn't that still use the profile provider?

Reply:
> What about running locally? Shouldn't that still use the profile provider?

Yep - https://github.com/guardian/workflow-frontend/pull/216/files#diff-2b19a54750b96913359e3934504c2b0dR16

Reply:
GH doesn't render that link very obviously. An image of the diff maybe?

![image](https://user-images.githubusercontent.com/836140/82649581-5739bc00-9c11-11ea-9631-83a3e1c1de2e.png)

Reply:
ohh, I see. I thought that AWS was _AWS_